docs/datastores/quickstart.ipynb

```diff
@@ -83,28 +93,88 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.07/lib/python3.10/site-packages/access_nri_intake/source/utils.py:37: UserWarning: Time coordinate does not include bounds information. Guessing start and end times.\n",
+      "/home/120/mcw120/access-nri/access-nri-intake-catalog/src/access_nri_intake/source/utils.py:113: UserWarning: Time coordinate does not include bounds information. Guessing start and end times.\n",
```

Comment:
This is potentially the most inane comment I've ever made, but I think we might want to rebuild the docs here after the release so that the warnings point to the right path - not your home dir.